Sources/NIO/EventLoop.swift

```diff
@@ -353,10 +359,18 @@ extension TimeAmount {
     public static func + (lhs: TimeAmount, rhs: TimeAmount) -> TimeAmount {
         return TimeAmount(lhs.nanoseconds + rhs.nanoseconds)
     }
+    
+    public static func += (lhs: TimeAmount, rhs: TimeAmount) -> TimeAmount {
```

Comment:
This function signature doesn’t seem to match what `AdditiveArithmetic` wants: that takes the argument _inout_.

Reply:
Added the _inout_ argument